.github/workflows/create-release-pr.yml

```diff
@@ -130,47 +141,25 @@ jobs:
         with:
           message: "bump up version to ${{ env.RELEASE_TAG }}"
 
-      - name: Create Pull Request to Master
+      - name: Create Pull Request to ${{ inputs.base_branch }}
+        if: ${{ env.PRE_RELEASE_VERSION == '' }}
```

Comment:
If we don't create a PR for pre_release_version, when/how do we use pre_release_version?

Reply:
Removed all pre-release support in latest update